Symfony/CS/Fixer/Symfony/ScalarCastFixer.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+/*
+ * This file is part of the PHP CS utility.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Symfony\CS\Fixer\Symfony;
+
+use Symfony\CS\AbstractFixer;
+use Symfony\CS\Tokenizer\Tokens;
+
+/**
+ * @author SpacePossum
+ */
+final class ScalarCastFixer extends AbstractFixer
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function fix(\SplFileInfo $file, $content)
+    {
+        $tokens = Tokens::fromCode($content);
+
+        for ($index = 0, $count = $tokens->count(); $index  < $count; ++$index) {
+            if (!$tokens[$index]->isCast()) {
+                continue;
+            }
+
+            $castFrom = substr($tokens[$index]->getContent(), 1, -1);
+            $castFrom = strtolower(trim($castFrom));
+            switch ($castFrom) {
+                case 'boolean' :
+                    $newContent = '(bool)';
+                    break;
+                case 'integer' :
+                    $newContent = '(int)';
+                    break;
+                case 'double' :
+                case 'real' :
+                    $newContent = '(float)';
+                    break;
+                default :
+                    continue 2;
```

Comment:
>`hack x2`

?

About the casing; I like this better because the fixer does not rely on the `LowerCaseCastFixer`, making it useful without it.